### ISSUE TITLE
Add a new CEDS element for the IHE Location 

### ISSUE THREAD

**Issue by manndoris-CEPI** [CEDStandards/CEDS-Elements#580]
[CEDS OSC Issue_106.docx](https://github.com/CEDStandards/CEDS-Elements/files/10209465/CEDS.OSC.Issue_106.docx)
This is for capturing needs not currently supported by the CEDS model.  Please do not send or share actual data as examples in this issue or in attachments. 

**Author(s)**

**Authoring Organization(s)**

**Email address**

**Use Case Title**
Provide a concise description that defines the use case.

**Use Case Description** 
List the proposed element name(s), definition(s), and/or option set(s).

**Use Case Background**
Provide information related to why these changes/additions are needed.

**Location of Element in the Domain Entity Schema**

**Comment by 4pins** [comment]
I'd comment that more information is need, but the proposal seems to cover that.  "Eastings and northings using a projected coordinate system with units often in meters (can be feet, etc). Not sure what the exact definition is. Found this one from ArcGIS."  LOL

**Comment by manndoris-CEPI** [comment]
Yes, definitely needs more discussion and input from other stakeholders


Doris Mann
CEPI Data Specialist

**Comment by manndoris-CEPI** [comment]
Additional info found on ArcGIS. We will need to have 2 elements for the required 2 coordinates. Do we also need to have an element that identifies what type of coordinates are used? 

https://gis.stackexchange.com/questions/260672/difference-between-x-y-and-latitude-longitude-or-meters

- "To add a table of x,y coordinates to your map, the table must contain two fields: one for the x-coordinate and one for the y-coordinate. The values in the fields may represent any coordinate system and units such as latitude and longitude or meters."
- For coordinates captured using a GPS, or by any means, longitude is the X value and latitude is the Y value. These are for a geographic coordinate system and have units of degrees.
- Alternatively your GPS may be set to return values for eastings and northings using a projected coordinate system with units often in meters (can be feet, etc).

**Comment by manndoris-CEPI** [comment]
Alternatively, we could move the latitude and longitude elements from K12 into PSC and call it good. :)
